src/App.js

```diff
@@ -26,6 +27,9 @@ function App() {
                     <Route exact path="/">
                         <LoginComponent />
                     </Route>
+                    <Route path="/processes">
+                        <RunningProcesses />
+                    </Route>
```

Comment:
This is very temporary. I just wanted to throw the `RunningProcesses` component in somewhere. I imagine that the `RunningProcesses` component will be axed soon since it's just a playground where you can interact with the new functionality that this PR introduces.